drafts/developer-guide/pages/data-access-layer/module-config.adoc

```diff
@@ -0,0 +1,38 @@
+include::ROOT:partial$variables.adoc[]
+
+= Compose Module configuration.
+
+To configure your database connection at module level, navigate to your {PRODUCT_NAME} instance ( http://qc.cortezaproject.org/) and click the *Low Code application*.
```

Comment:
`http://qc.cortezaproject.org/` will usually not be the case for other users; consider doing something along the lines of:

`... navigate to your {PRODUCT_NAME} instance, such as {LOCAL_HOSTNAME}, and click on ...`

The sentence above indicates you're providing an example and not necessarily the absolute truth.